### ISSUE TITLE
Use of Angular Material Side Navbar module will reduce code

### ISSUE THREAD

**Issue by lunayach** [systers/PC-Prep-Kit#101]
## Description
As a coder,
I need to use the existing APIs,
so that the code becomes cleaner and efficient.

## Mocks
We are writing the toggle() method to hide and show and writing explicitly the animations for changing the state. 

### Update 
- [ ] Use of Angular Material Side Navbar module. i.e. {MatSidenavModule} in Navabar component.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
30 minutes

**Comment by shreyans29** [comment]
Hey @lunayach , You can work on this Issue, but I see the estimation is way below the actual time it might take. Once you change the orientation of other components may change and you will have to fix them also. Any way go ahead with this. Send a PR when done.

**Comment by shreyans29** [comment]
Also Please update the issue, and put the statement you have put in mocks below description. Mocks are supposed to be graphical representations of what the end result might look like.

**Comment by lunayach** [comment]
@shreyans29 Yes I also realize the same :+1:

**Comment by lunayach** [comment]
@shreyans29 as my previous PR is not merged, shall I have to make a new branch and commit changes for both the PR's(i.e. this one and the earlier unmerged PR)  and then do the PR.

**Comment by shreyans29** [comment]
@lunayach  No, your new PR should only contain the changes for this issue.
